app/views/exams/StartPage.vue

```diff
@@ -80,6 +75,16 @@ export default {
       const clans = me.get('clans') || []
       return clans.includes(this.exam?.clanId)
     },
+    buttonValue () {
+      if (this.newUser === NEW_USER) {
+        return $.i18n.t('exams.start_exam')
+      } else {
+        return $.i18n.t('exams.continue_exam')
+      }
+    },
+    limitedDuration () {
+      return this.userExam?.duration || this.exam?.duration
```

Comment:
with new logic, wouldn't all userExam have duration, do we need fallback on exam?

Reply:
no. for the first userExam we can skip the duration (so that do not set unnecessary duplicated `duration` for every user)
the duration is actually needed only if the old userExam set an `extraDuration` and copy that. (use 2 keys so that we wouldn't go into infinite loop of creating new user exams)